distribution/changelog.txt

```diff
@@ -1187,6 +1182,11 @@ This is the first fully implemented version of OpenRCT2. RCT2.EXE is no longer r
 - Fix: [#3063] Search exe directory for SSL bundle as well as CWD.
 - Fix: [#3120] Negative cash in finance window is not red.
 - Fix: [#4134] Can’t enable park-wide photo price for log flume and river rapids.
+- Technical: lodepng dropped in return for libpng.
+- Technical: SDL2 upgraded from 2.0.3 to 2.0.4.
+- Technical: argparse dropped in return for bespoke command line parsing implementation.
+- Technical: Integrated breakpad for (manual) crash reporting.
+- Removed: Anti-cheat code that detected money hack attempts.
```

Comment:
Order here is still wrong.

Reply:
Still incorrect.

Reply:
I closed it too early because I have the change ready, just not pushed yet because I want to clear up the history and rebase it.